### ISSUE TITLE
[a11y-defect-2] Keyboard: New evidence request views must set focus on H1 instead of breadcrumbs

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#108619]
### Point of contact

@1copenut

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

This issue came about while doing pre-staging review accessibility testing for BMT II's third-party evidence request improvements. I noticed the focus was being set on the breadcrumbs instead of the H1 when the Status of your claims view and claim detail views were loaded. This reads a lot of extra information to screen readers and causes keyboard users to have to click through a number of breadcrumbs before they move through the page.

Screenshots attached.

---
![Image](https://github.com/user-attachments/assets/82115435-5248-44b0-ae6b-69efe9d0b7ce)

---
![Image](https://github.com/user-attachments/assets/a66fd4d9-3498-41df-9650-7a9b31cdb6a1)

### Reproduction steps

This issue is currently behind feature flag awaiting promotion to staging. When it's promoted, here's the steps to reproduce.

1. Log into staging.va.gov with a known test user, preferably 211.
2. Change your URL to `/track-claims/your-claims` and wait for the page to load
3. Click on a claim "Detail" link. Verify focus is on the breadcrumb when the page loads.
4. Now click on Files, and click one of the "About this notice" links in the blue (third-party) evidence requests. Verify focus is also on the breadcrumbs.

### Proposed solution or next steps

BMT II team thinks there's a helper function overriding default behavior that will be removed and should fix this issue.

### References, articles, or WCAG support

1. [Understanding SC 1.3.2 (Meaningful Sequence): Level A](https://www.w3.org/WAI/WCAG22/Understanding/meaningful-sequence.html)

### Type of issue

- [ ] Axe-core
- [x] Screenreader | assistive tech & device support
- [x] Keyboard
- [ ] Zoom
- [ ] Color, typography, & visual elements
- [ ] Components and pattern standards
- [ ] Content organization
- [ ] Focus
- [ ] Headings
- [ ] Content style guide
- [ ] Link and button labels
- [ ] Semantic HTML
- [ ] Markup and metadata
- [ ] Non-text content and media
- [ ] Overall user experience
- [ ] Cognitive | user flows & navigating
- [ ] URL standards
- [ ] Something else

**Comment by 1Copenut** [comment]
@salihaghaffar This one is working correctly. Good to move to done.
